doc/source/whatsnew/v1.5.0.rst

```diff
@@ -125,6 +125,7 @@ Other enhancements
 - Added ``validate`` argument to :meth:`DataFrame.join` (:issue:`46622`)
 - A :class:`errors.PerformanceWarning` is now thrown when using ``string[pyarrow]`` dtype with methods that don't dispatch to ``pyarrow.compute`` methods (:issue:`42613`)
 - Added ``numeric_only`` argument to :meth:`Resampler.sum`, :meth:`Resampler.prod`, :meth:`Resampler.min`, :meth:`Resampler.max`, :meth:`Resampler.first`, and :meth:`Resampler.last` (:issue:`46442`)
+- Added validation of boolean kwargs in string series and DataFrame methods, along with tests (:issue:`16714`)
```

Comment:
pls be very specific on which methods / kwargs are updated here a reader is not going to find this note useful.

Reply:
i would also put this in the appropriate bug fix categories (even if you do multiple notes is ok)